Control/test/lib/mocha-control-service.js

```diff
@@ -23,33 +23,27 @@ const AssertionError = require('assert').AssertionError;
 
 describe('Control Service test suite', () => {
   describe('Check Constructor', () => {
-    it('should throw error due to missing PadLock dependency', () => {
```

Comment:
As these should comeback with the future ticket maybe it helps if we skip the for now rather than deleting
`it.skip`

Reply:
This will need to be totally reimplemented anyway